modules/ztp-things-to-consider-when-creating-custom-resource-policies.adoc

```diff
@@ -0,0 +1,12 @@
+Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-things-to-consider-when-creating-custom-resource-policies_{context}"]
+= Things to consider when creating custom resource policies
```

Comment:
s/ Things to consider/ Considerations when creating...

Reply:
Changed.